rust/cbindgen.toml

```diff
@@ -0,0 +1,6 @@
+autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */"
+language = "C"
+header = "#include <gio/gio.h>\ntypedef GError RORGError;"
```

Comment:
This is definitely a hack, I need to look more at cbindgen and figure out if we can tell it that we depend on `GError` externally on both sides.